frontend/views/components/Avatar.vue

```diff
@@ -82,6 +82,8 @@ export default ({
   },
   watch: {
     src (to) {
+      // NOTE: src could be null while logging out
+      //       since typeof null === 'object', we should check if it's falsy
       if (to && typeof to === 'object') {
```

Comment:
It seems like someone already fixed this issue. After merging the latest master branch, I could see it's already fixed. So I have just added comment. The previous code was like below.

**Previous**
```
if (typeof to === 'object')
```